projects/vdk-core/plugins/build-plugin.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -e
+
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+PLUGIN_NAME=$(basename "$(pwd)")
+echo "Building plugin $PLUGIN_NAME"
+
+PIP_EXTRA_INDEX_URL=${PIP_EXTRA_INDEX_URL:-'https://pypi.org/simple'}
+
+pip install -U pip setuptools pre-commit
+pre-commit install
+
+pip install --extra-index-url $PIP_EXTRA_INDEX_URL -r requirements.txt
+pip install -e . --extra-index-url $PIP_EXTRA_INDEX_URL
+
+# List exceptions to below check here.
+# Those are not technically plugins so they would not have entry point defined.
+if [ "$PLUGIN_NAME" != 'quickstart-vdk' ] && [ "$PLUGIN_NAME" != 'vdk-test-utils' ]; then
```

Comment:
This is probably fine for now, but we should think about a programmatic way of separating plugins from supplementary packages like test-utils. Also, what about taurus-vdk?

Reply:
I plan to delete taurus-vdk. It is not necessary here